lib/kubernetes-deploy/deploy_task.rb

```diff
@@ -449,22 +452,35 @@ def log_pruning(kubectl_output)
       @logger.summary.add_action("pruned #{pruned.length} #{'resource'.pluralize(pruned.length)}")
     end
 
-    def record_apply_failure(err)
+    def record_apply_failure(err, resources: [])
       warn_msg = "WARNING: Any resources not mentioned in the error(s) below were likely created/updated. " \
         "You may wish to roll back this deploy."
       @logger.summary.add_paragraph(ColorizedString.new(warn_msg).yellow)
 
       unidentified_errors = []
+      sensitive_filenames = resources.select(&:kubectl_output_is_sensitive?).map { |r| File.basename(r.file_path) }
```

Comment:
nit: `sensitive_filenames` -> `filenames_with_sensitive_content`

Reply:
Addressed in https://github.com/Shopify/kubernetes-deploy/pull/438